### PR TITLE
Pa/resolve remaining metadata issues

### PR DESCRIPTION
**Pull request by azunre**

### CODE REVIEW

File: DukeD3MWrapper/wrapper.py [hunk 1]
```diff
@@ -167,8 +170,31 @@ def produce(self, *, inputs: Inputs, timeout: float = None, iterations: int = No
...
+        duke_df = d3m_DataFrame(out_df_duke)
+        # first column ('subject tags')
+        col_dict = dict(duke_df.metadata.query((metadata_base.ALL_ELEMENTS, 0)))
+        col_dict['structural_type'] = type("it is a string")
+        col_dict['name'] = "subject tags"
+        col_dict['semantic_types'] = ('http://schema.org/Text', 'https://metadata.datadrivendiscovery.org/types/Attribute')
+        duke_df.metadata = duke_df.metadata.update((metadata_base.ALL_ELEMENTS, 0), col_dict)
+         # second column ('confidences')
+        col_dict = dict(duke_df.metadata.query((metadata_base.ALL_ELEMENTS, 1)))
+        col_dict['structural_type'] = type("1.0")
```
Comment (cdbethune) [inline DukeD3MWrapper/wrapper.py]: Given that this will yield a type of `str`, is the `confidences` column an `obj` in the Dataframe?

Comment (azunre) [reply DukeD3MWrapper/wrapper.py]: yes, looks like it:
```>>> print(result.value.dtypes)
d3mIndex        object
cylinders       object
displacement    object
horsepower      object
weight          object
acceleration    object
model           object
origin          object
class           object
subject tags    object
confidences     object
dtype: object
>>> 
```

Comment (cdbethune) [reply DukeD3MWrapper/wrapper.py]: Good from my end then.

**Comment by azunre** [comment]
There was no real consensus discovered from TA2s on whether to append, prepend, etc. As such, I am not going to bother with it for now (will participate in that working group moving forward) -- but for now duke will return just the output d3m dataframe, with appropriate metadata...

**Comment by azunre** [comment]
see #ta1ta2 channel on d3m slack for more details, merging...

**Pull request merged**